### ISSUE TITLE
Uncaught Error: Cannot call function 'getPosition' on missing remote object 220Error: Cannot cal...

### ISSUE THREAD

**Issue by Kpraful** [atom/atom#21178]
Note: I am not fully aware what triggered this issue. But this is not the first time I am having this.
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.40.1 x64
**Electron**: 3.1.10
**OS**: Kali GNU/Linux
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: Cannot call function 'getPosition' on missing remote object 220
Error: Cannot call function 'getPosition' on missing remote object 220
    at throwRPCError (/usr/share/atom/resources/electron.asar/browser/rpc-server.js:145:17)
    at EventEmitter.<anonymous> (/usr/share/atom/resources/electron.asar/browser/rpc-server.js:367:7)
    at EventEmitter.emit (events.js:182:13)
    at WebContents.<anonymous> (/usr/share/atom/resources/electron.asar/browser/api/web-contents.js:294:13)
    at WebContents.emit (events.js:182:13)

```
At /usr/share/atom/resources/electron.asar/renderer/api/remote.js:219

Error: Cannot call function 'getPosition' on missing remote object 220
Error: Cannot call function 'getPosition' on missing remote object 220
    at throwRPCError (/usr/share/atom/resources/electron.asar/browser/rpc-server.js:145:17)
    at EventEmitter.<anonymous> (/usr/share/atom/resources/electron.asar/browser/rpc-server.js:367:7)
    at EventEmitter.emit (events.js:182:13)
    at WebContents.<anonymous> (/usr/share/atom/resources/electron.asar/browser/api/web-contents.js:294:13)
    at WebContents.emit (events.js:182:13)

    at metaToException (/usr/share/atom/resources/electron.asar/renderer/api/remote.js:271:17)
    at Object.exception (/usr/share/atom/resources/electron.asar/renderer/api/remote.js:219:30)
    at metaToValue (/usr/share/atom/resources/electron.asar/renderer/api/remote.js:223:28)
    at Object.remoteMemberFunction (/usr/share/atom/resources/electron.asar/renderer/api/remote.js:123:16)
    at AtomEnvironment.getWindowDimensions (/usr/share/atom/resources/app/static/<embedded>:1:705395)
    at AtomEnvironment.storeWindowDimensions (/usr/share/atom/resources/app/static/<embedded>:1:705810)
    at WindowEventHandler.handleWindowBlur (/usr/share/atom/resources/app/static/<embedded>:11:285354)
```

### Commands

```
  3x -3:45 core:move-right (input.hidden-input)
     -3:43.9.0 editor:newline (input.hidden-input)
  5x -3:41.9.0 core:backspace (input.hidden-input)
     -3:36.6.0 core:save (input.hidden-input)
  5x -2:52.4.0 core:backspace (input.hidden-input)
     -2:48.8.0 core:move-down (input.hidden-input)
  2x -1:15.9.0 core:undo (input.hidden-input)
  2x -1:13.5.0 core:redo (input.hidden-input)
     -1:12 core:save (input.hidden-input)
     -1:02.2.0 core:undo (input.hidden-input)
     -0:55.3.0 core:backspace (input.hidden-input)
  5x -0:54.8.0 core:move-right (input.hidden-input)
     -0:53.2.0 core:backspace (input.hidden-input)
     -0:52.8.0 core:save (input.hidden-input)
     -0:48.8.0 core:backspace (input.hidden-input)
     -0:48.3.0 core:save (input.hidden-input)
```

### Non-Core Packages

```

```

**Comment by sadick254** [comment]
Thanks for contributing!

All new issues now require steps to reproduce (also called "repro steps") the problem or the behavior you wish to see in Atom. We do this so we can be certain that we are understanding what it is you're describing and don't prematurely close the issue or implement the wrong feature. If you need help writing detailed repro steps, please see the [How Can I Contribute? section of the Atom CONTRTIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-can-i-contribute).

**Comment by sadick254** [comment]
Thank you @Kpraful for opening this issue. This was issue present in electron versions 4 and below. The latest atom version `v1.51.0` uses electron version `5.0.3`. The issue was fixed on this commit https://github.com/electron/electron/commit/a8f2646ba682d65f94ea628ac11b1a5521bcedcd. The fix has been available from electron version `5.0.0`. I would advice you to upgrade to the latest atom version. I am closing this for now but happy to reopen if you still experience the issue.
